buildSrc/src/main/java/org/opensearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -298,47 +292,7 @@ private static int findDefaultParallel(Project project) {
         // Since it costs IO to compute this, and is done at configuration time we want to cache this if possible
```

Comment:
Remove this comment line? Otherwise LGTM.

Reply:
thanks, indeed this is outdated now.